src/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.4.2-dev'
\ No newline at end of file
+__version__ = '0.4.3-dev'
```

Comment:
dont change version

Reply:
Will build with current version in the future